apispec/utils.py

```diff
@@ -98,4 +98,4 @@ def validate_swagger(spec):
                 stderr=subprocess.STDOUT,
             )
         except subprocess.CalledProcessError as error:
-            raise exceptions.SwaggerError(error.output.decode('utf-8'))
+            raise exceptions.PluginError(error.output.decode('utf-8'))
```

Comment:
I'm just not really sure if swagger mentions should be somewhere but in `ext/marshmallow/swagger.py`.